src/user/validate_user.js

```diff
@@ -0,0 +1,19 @@
+const debug = require('debug')('user') // eslint-disable-line
+
+/**
+ * validateUser - validate the contents of a user object before saving
+ *
+ * @param  {object} user data from web form or sms endpoint
+ * @return {string|undefined}
+ *    returns an error string if the user is invalid
+ *    returns undefined if user is valid
+ */
+module.exports = function validateUser(user) {
+  if (!user) return 'user must exist'
+
+  if (!user.name) return 'user must have a name'
+  if (!user.zip) return 'user must have a zip code'
```

Comment:
this will break the web form until #23 is complete, but i'll get that done as soon as #31 is merged.